examples/README.md.tsx

```diff
@@ -45,8 +45,9 @@ const Readme: Component = () => (
     <Heading level={2}>Installation</Heading>
     Add <InlineCode>jsx-readme</InlineCode> to your{" "}
     <InlineCode>devDependencies</InlineCode> and install it. I recommend using
-    it with <InlineCode>ts-node</InlineCode>. Then all you need to do is add a
-    file like in the example below and run it via{" "}
+    it with <InlineCode>ts-node</InlineCode>. Then all you need to do is run {" "} <InlineCode>npx jsx-readme init [destinationPath]</InlineCode>
+    where the {" "} <InlineCode>[destinationPath]</InlineCode> is where the {" "} <InlineCode>README.md.tsx</InlineCode> file will be outputed and once done
+    you run the {" "} <InlineCode>README.md.tsx</InlineCode> via{" "}
```

Comment:
Can we simplify this further by doing the following when running the init script:
* If `ts-node` is neither a dependency, nor devDependency, add it to the devDependencies
* If there is no `build:readme` script in scripts, add a new `build:readme` script that runs the readme file with `ts-node`
* Instead of asking for the location of the new file, put it into the CWD where also package.json is located (and fail the script if there is no package.json with a helpful error message

Reply:
Understood! :ok_hand: